src/test/java/api/requests/RequestsAPILoanRenewalTests.java

```diff
@@ -171,4 +192,46 @@ public void allowOverrideRenewalLoanByBarcodeWhenFirstRequestInQueueIsHold() thr
 
     assertThat(response.getJson().getString("action"), is("renewedThroughOverride"));
   }
+
+  @Test
+  public void forbidRenewalOverrideWhenFirstRequestIsNotRecall()
+    throws MalformedURLException,
+    InterruptedException,
+    TimeoutException,
+    ExecutionException {
+
+    final InventoryItemResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
+    final IndividualResource rebecca = usersFixture.rebecca();
+
+    DateTime loanDueDate = new DateTime(
+      2018, APRIL, 21,
+      11, 21, 43
+    );
+
+    loansFixture.checkOutByBarcode(smallAngryPlanet, rebecca, loanDueDate);
+
+    useLoanPolicyAsFallback(
```

Comment:
Is this different to the default set up by the API tests?

Reply:
It's the same. Do you think it could be omitted?

Reply:
Given it is the same, I think it can be omitted from this test (and any other that may also do this)

Reply:
Done